source/src/molecular-unfolding/cdk/stack-main.ts

```diff
@@ -50,6 +51,13 @@ export class MainStack extends SolutionStack {
     this.setDescription(DESCRIPTION);
     const stackName = this.stackName.replace(/[^a-zA-Z0-9_]+/, '').toLocaleLowerCase();
 
+    new CfnRule(this, 'deploymentRegionRules', {
+      assertions: [{
+        assert: Fn.conditionContains(['us-west-2', 'us-east-1'], this.region),
+        assertDescription: 'deployment regions',
```

Comment:
use informative description, for example, `supported regions are us-east-1, us-west-2`

Reply:
changed the Description